api/spec/integration/api/v2/platform/menus_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'swagger_helper'
+
+describe 'Menus API', swagger: true do
+  include_context 'Platform API v2'
+
+  resource_name = 'Menu'
+  include_example = 'menu_items'
+  filter_example = 'location=header'
```

Comment:
```suggestion
  filter_example = 'location_eq=header'
```
https://github.com/activerecord-hackery/ransack#search-matchers